static/app/views/settings/components/teamSelect/teamSelectForMember.tsx

```diff
@@ -110,21 +111,27 @@ function TeamSelect({
 
   return (
     <Panel>
-      <PanelHeader hasButtons>
-        {t('Team')}
-        <DropdownAddTeam
-          disabled={disabled}
-          isLoadingTeams={isLoadingTeams}
-          isAddingTeamToMember
-          canCreateTeam={false}
-          onSearch={onSearch}
-          onSelect={onAddTeam}
-          onCreateTeam={onCreateTeam}
-          organization={organization}
-          selectedTeams={selectedTeams.map(tm => tm.slug)}
-          teams={teams}
-        />
-      </PanelHeader>
+      <TeamPanelHeader>
+        <div>{t('Team')}</div>
+        <div />
+        <div>
+          <TeamRoleColumnLabel />
+        </div>
+        <div>
+          <DropdownAddTeam
+            disabled={disabled}
+            isLoadingTeams={isLoadingTeams}
+            isAddingTeamToMember
```

Comment:
I think this is undefined - it defaults to `false` so we can probably just take it out
![Screenshot 2023-07-14 at 7 14 07 AM](https://github.com/getsentry/sentry/assets/26236981/ef7904dc-b0bb-45c1-a77f-f9dae4c6354c)

Reply:
This is equivalent to `isAddingTeamToMember={true}`.